src/main/resources/view/PersonListCard.fxml

```diff
@@ -25,21 +25,21 @@
             <Region fx:constant="USE_PREF_SIZE" />
           </minWidth>
         </Label>
-        <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
-        <HBox fx:id="meritScoreHBox">
-          <Label text="(Merit Score: " styleClass="cell_big_label"/>
+        <Label fx:id="name" text="\$first" styleClass="cell_big_label" wrapText="true"/>
+        <HBox fx:id="meritScoreHBox" alignment="CENTER_RIGHT" minWidth="200">
+          <Label text="(Merit Score: " styleClass="cell_big_label" wrapText="true"/>
```

Comment:
Changes were made to allow for wrap text because our commands require the use of the full name as a unique identifier. Without the full name, library managers cannot identify the different users when the difference in name spelling is truncated, hence it is a functionality bug that needs to be fixed.